src/modals/_ApplicataionSwitcherModal.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import List, { Item } from 'terra-list';
+import Hyperlink from 'terra-hyperlink';
+import ApplicationModal from '@cerner/terra-application/lib/application-modal/ApplicationModal';
```

Comment:
100% curious, is it possible to use this import?

```suggestion
import ApplicationModal from '@cerner/terra-application/lib/application-modal';
```

Reply:
In the future, yes probably, but as of now we haven't added the index to the folder to import from.